applications/openshift/api-server/api_server_token_auth/oval/shared.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="api_server_token_auth" version="1">
+    <metadata>
+      <title>Disable Token-based Authentication</title>
+      <affected family="unix">
+        <platform>multi_platform_ocp</platform>
+      </affected>
+      <description>token-auth-file should not be configured on the master node.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion comment="token-auth-file does not exist" test_ref="test_api_server_token_auth" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="none_exist" comment="token-auth-file does not exist" id="test_api_server_token_auth" version="1">
+    <ind:object object_ref="object_api_server_token_auth" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_api_server_token_auth" version="1">
+    <ind:filepath>/etc/origin/master/master-config.yaml</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*token-auth-file\:[\n]+[\s*]-[\s]*(\S+)[\s]*$</ind:pattern>
```

Comment:
Should this have ``servingInfo``, as the other API Server OVALs do?

Reply:
No because the evaluation is that `token-auth-file` does not exist at all.

Reply:
fair enough :)